src/main/kotlin/pl/put/smartgarden/domain/device/SensorType.kt

```diff
@@ -0,0 +1,5 @@
+package pl.put.smartgarden.domain.device
+
+enum class SensorType {
+    IRRIGATION, ILLUMINANCE, HUMIDITY, TEMPERATURE
```

Comment:
Nawiązując do poprzedniego komentarza dotyczącego jednostki (°C), nie podoba mi się to, że jednostka nie jest zawarta w "sensorType".